frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  type Control,
+  type FieldErrors,
+  type FieldPath,
+  type FieldValues,
+  useController,
+} from 'react-hook-form';
+
+import styles from './styles.module.scss';
+
+type Properties<T extends FieldValues> = {
+  control: Control<T, null>;
+  errors: FieldErrors<T>;
+  label?: string;
+  name: FieldPath<T>;
+  className?: string;
+  labelClassName?: string;
+};
+
+const Checkbox = <T extends FieldValues>({
```

Comment:
Let's name it toggle-checkbox or switch-checkbox - as it actually looks toggle/switch